src/content/smart-bulletins/038.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Smart Bulletin No. 038"
+intro: "Notice Regarding Unenforceability of Unauthorized Obligations for Task Orders Awarded Under GSA SmartPay® 3 (SP3)"
+number: "No. 038"
+order: 038
+businessline: "Purchase, Travel, Fleet, Integrated"
+published: "Sep 26, 2023"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| September 26, 2023 | Implementation of Smart Bulletin #38 |
+
+## Business Line(s) Affected
+
+ Purchase, Travel, Fleet, Integrated
+
+## Introduction
+
+The purpose of this Smart Bulletin is to provide clarity for customer agencies with
```

Comment:
The purpose of this Smart Bulletin is to provide clarity for customer agencies with task orders awarded under the SP3 master contracts. If any government-authorized end user clicks on an “I agree” click box or other comparable mechanisms within the servicing bank system(s), this action may not automatically bind the government to the agreement. This includes any “I agree” or similar text that might be found in card management systems, including, but not limited to, the Electronic Access Systems of the banks (US Bank and Citibank) and other systems such as MasterCard’s Insights on Demand (IOD) system and Visa’s Intellilink system.